src/pages/install.astro

```diff
@@ -0,0 +1,179 @@
+---
+import i18next, { t, changeLanguage } from "i18next"
+import Base from "../layouts/Base.astro"
+
+changeLanguage("en")
+
+const title = `Install | ${t("site.title")}`
+const description = t("site.description")
+---
+<!-- All changes (including styles) are limited to this page since it is temporary -->
+
+<Base title={title} description={description}>
+  <section class="section">
+    <div class="contentWrapperSm">
+      <img
+        alt="Web Monetization Icon"
+        class="wm-icon"
+        src="/img/wm-logo--animated.svg"
+        width="120"
+        height="86"
+      />
+      <h1 class="heading2">Web Monetization</h1>
+    </div>
+  </section>
+  <section>
+    <div class="contentWrapperLg">
+      <div class="cards twoColGrid">
```

Comment:
`.cards` style says `display: flex`, but `.twoColGrid` is grid..

Let's add a `justify-items: center;` either way.

Reply:
Can I suggest a counter-suggestion, of simply removing the class `twoColGrid`? `justify-items: center` is not necessary in this instance.